rasa/utils/io.py

```diff
@@ -151,3 +151,13 @@ def write_yaml_file(data: Dict, filename: Text):
     """
     with open(filename, "w") as outfile:
         yaml.dump(data, outfile, default_flow_style=False)
+
+
+def is_in_subdirectory(path: Text, potential_parent_directory: Text) -> bool:
```

Comment:
for some reason i found the `in` confusing
```suggestion
def is_subdirectory(path: Text, potential_parent_directory: Text) -> bool:
```